### PR TITLE
OD-786 EF does not import chapters when the stories are contained in db

### PR DESCRIPTION
**Pull request by brianna-dardin**

### CODE REVIEW

File: efiction/tests/test_data/storiesef/5/842.txt [hunk 1]
```diff
@@ -1,3 +1,3 @@
 This story was originally Windows 1252.
 
-E�s in ipsum oc�rr�ret. Al�i vidiss� �ffic�endi v�m at, �d ridens gr�ec� detraxit vis. Legim�s vol�tp�t �ompr�hensam �um ne, liber fe�gait �ffic�antur q�� an, usu in v��ib�s sal�tat�s. Nec eu justo tritani qu��sti�. V�l amet m�lorum �bhorr�ant �t, v�lit med�ocritat�m �x pro. Mu���s p�ricula ne sea, �am mand�m�s ass�nti�r an, sed �t �q��d�m ornatus.
\ No newline at end of file
+Eôs in ipsum ocûrrëret. Alïi vidissé éfficïendi vïm at, àd ridens gràecï detraxit vis. Legimùs volûtpât çomprèhensam éum ne, liber feùgait ëfficîantur qùî an, usu in vôçibûs salùtatùs. Nec eu justo tritani quâèstiœ. Vël amet màlorum àbhorréant ât, vëlit medïocritatëm éx pro. Muçîùs pêricula ne sea, èam mandàmûs asséntiôr an, sed ët êqûîdêm ornatus.
```
Comment (ariana-paris) [inline efiction/tests/test_data/storiesef/5/842.txt]: I think an oopsie has happened to this Windows 1252 files as all the diacritics have been trashed. Maybe you can revert the change just to this file and hope it doesn't get saved again with the wrong encoding by whatever trashed it?

**Review by ariana-paris — CHANGES_REQUESTED** [review]
OMG, tests and everything ❤️  There's just one problem where the encoding of a Windows 1252 file has been changed, trashing all the non-ASCII characters (I... guess it isn't used in a test but might as well preserve its encoding if we ever need one of these files!). But aside from that it looks good. Thank you very much for working on this!

File: efiction/tests/test_data/storiesef/5/842.txt (hunk 1 above)
Comment (ariana-paris) [reply efiction/tests/test_data/storiesef/5/842.txt]: Weirdly, I now notice that GitHub.com is showing me this diff with and without the correct characters, so it could be that this is actually a GitHub problem rather than a problem with the file. But worth checking as I don't think it was part of your changes anyway. 🤔

Comment (brianna-dardin) [reply efiction/tests/test_data/storiesef/5/842.txt]: This is actually a purposeful change. When test_chapters_no_text_processing runs, it actually does read all the files in this directory, and this file triggers the too many warnings exception when the encoding is left as Windows 1252. I converted it to UTF-8 to bypass the error. I'm open to other ideas to bypass the error (mocking the open method? refactoring it to automatically attempt other encoding types?) if we need to keep the original encoding.

ETA: I see the issue you're mentioning, the characters are correct on the main PR link but not on the files changed tab. They look fine when I view the file on Notepad++ and also when [viewing the file on the branch](https://github.com/otwcode/open-doors-eFiction/blob/od786/efiction/tests/test_data/storiesef/5/842.txt), so I think it's ok, just GitHub being weird.

Comment (ariana-paris) [reply efiction/tests/test_data/storiesef/5/842.txt]: (comments in Slack for better interactivity)

**Review by ariana-paris — APPROVED** [review]
Not sure why Github UI thinks the problem file has changed but it appears to still be Windows-1252 so let's go with this. Thank you so much for pegging away at it and getting testing working! Future us will be very grateful...
